### PR TITLE
CIT-41 #time 5h Update CRUD APIs from APIView to ModelViewSet in blog…

### PR DESCRIPTION
**Pull request by codal-uchoudhary**
… project

### CODE REVIEW

File: Projects/My_website/myWebsite/blog/urls.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 from rest_framework.routers import DefaultRouter
 from . import views
+from rest_framework import routers
 
 
+router = routers.DefaultRouter()
+router.register(r"", views.myBlog)
+
+router2 = routers.DefaultRouter()
+router2.register(r"", views.blog)
```
Comment (arvish-codal) [inline Projects/My_website/myWebsite/blog/urls.py]: @codal-uchoudhary 
why do we need 2 routers ?

Comment (codal-uchoudhary) [reply Projects/My_website/myWebsite/blog/urls.py]: because we have two ModelViewSet class (myBlog , blog)

Comment (arvish-codal) [reply Projects/My_website/myWebsite/blog/urls.py]: @codal-uchoudhary 
There's only 1 router per project. ALWAYS.
Make the changes and FIX the issues if you encounter any.

Comment (codal-uchoudhary) [reply Projects/My_website/myWebsite/blog/urls.py]: okay, let me change this

Comment (codal-uchoudhary) [reply Projects/My_website/myWebsite/blog/urls.py]: routers are updated

**Review by arvish-codal — APPROVED** [review]

**Pull request merged**